src/OrbitCaptureClient/GpuQueueSubmissionProcessor.cpp

```diff
@@ -212,7 +211,10 @@ void GpuQueueSubmissionProcessor::DeleteSavedGpuJob(int32_t thread_id,
   if (!tid_to_submission_time_to_gpu_job_.contains(thread_id)) {
     return;
   }
-  auto& submission_time_to_gpu_job = tid_to_submission_time_to_gpu_job_.at(thread_id);
+  // This method might be called if the "capture start" falls directly inside a GpuJob, and we thus
```

Comment:
I think "if" is slightly inaccurate (that's not the only case when this method is called). Maybe "even when" would be more precise?

Reply:
Done.